frontend/public/components/monitoring/receiver-forms/routing-labels-editor.tsx

```diff
@@ -48,89 +62,90 @@ export const RoutingLabelEditor = ({ formValues, dispatchFormChange, isDefaultRe
-      <p className="pf-v6-u-mb-md">
+    <FormSection
+      title={
+        <>
+          {t('public~Routing labels')}{' '}
+          {!isDefaultReceiver && (
+            <span className="pf-v6-c-form__label-required" aria-hidden="true">
+              {' '}
+              *
+            </span>
+          )}
+        </>
+      }
+    >
+      <Content component={ContentVariants.p} className="pf-v6-u-mb-0">
         <Trans ns="public">
           Firing alerts with labels that match all of these{' '}
-          <ExternalLink
-            href="https://prometheus.io/docs/alerting/latest/configuration/#matcher"
-            text={t('public~matchers')}
-          />{' '}
-          will be sent to this receiver.
+          <ExternalLink href={DOC_URL_PROMETHEUS_MATCHERS} text={t('public~matchers')} /> will be
+          sent to this receiver.
         </Trans>
-      </p>
+      </Content>
       {isDefaultReceiver && (
-        <Grid hasGutter className="form-group" key="default">
-          <GridItem span={10}>
-            <div className="form-group">
-              <span className="pf-v6-c-form-control pf-m-disabled">
-                <input
-                  type="text"
-                  data-test-id="label-default"
-                  value={DEFAULT_RECEIVER_LABEL}
-                  disabled
-                  required
-                />
-              </span>
-            </div>
-          </GridItem>
-        </Grid>
+        <FormGroup>
+          <Grid hasGutter key="default">
+            <GridItem span={10}>
+              <TextInput type="text" value={DEFAULT_RECEIVER_LABEL} isDisabled isRequired />
+            </GridItem>
+          </Grid>
+        </FormGroup>
       )}
       {_.map(formValues.routeLabels, (routeLabel, i: number) => {
         return (
-          <Grid hasGutter className="form-group" key={i}>
-            <GridItem span={10}>
-              <div className="form-group">
-                <span className="pf-v6-c-form-control">
-                  <input
-                    type="text"
-                    data-test-id={`label-${i}`}
-                    onChange={onRoutingLabelChange(i)}
-                    placeholder={t('public~Matcher')}
-                    value={routeLabel}
-                    required
-                  />
-                </span>
-              </div>
-            </GridItem>
-            <GridItem span={2}>
-              <Tooltip content={t('public~Remove')}>
-                <Button
-                  icon={<MinusCircleIcon />}
-                  type="button"
-                  onClick={() => removeRoutingLabel(i)}
-                  aria-label={t('public~Remove')}
-                  isDisabled={!isDefaultReceiver && formValues.routeLabels.length <= 1}
-                  variant="plain"
-                  data-test-id="remove-routing-label"
+          <FormGroup key={i}>
+            <Grid hasGutter>
+              <GridItem span={10}>
+                <TextInput
```

Comment:
The spacing between the `Matcher` labels is a little tall and we could make their width consistent with the rest of the form elements.

A better component structure for these Routing labels inputs that have associated buttons would be to use PF `InputGroup` and `InputGroupItem` within a `Stack` layout.

https://codesandbox.io/p/sandbox/gifted-shape-jhc3l4?file=%2Findex.tsx%3A11%2C3-12%2C18